packages/demo/package.json

```diff
@@ -64,6 +64,8 @@
     "@web3-onboard/xdefi": "^2.0.5",
     "@web3-onboard/zeal": "^2.0.3",
     "@web3-onboard/particle-network": "^2.0.1",
+    "@web3-onboard/passport": "^2.0.0-alpha.1",
+    "@0xpass/webauthn-signer": "2.0.0",
```

Comment:
I am unclear why this dep is required if its also part of the package.
Could the init object just be passed in as part of the passportModule it and `WebauthnSigner` be initialized within the package?

Reply:
@Adamj1232 We have multiple signer types, and support a specific signer interface, so this setup is setting up the flow where users are using passkeys / webauthn signer, so it is required in the application context of setting up the signer, and is the most convenient way of setting it up, however it is not required in the sense it should be included in the onboard package, in future would like to add other signer types as well, but just starting with passkeys, so setting it up this way makes it easier for us to be agnostic with the signer types

so in the demo we have the passport onboard package and the webauthn signer, but in the actual passport package we have the core passport sdk, as well as our viem client, so separate packages, but if you'd feel strongly you'd prefer the webauthn signer to be initialised in the passportModule, I can push that and do some refactoring when adding other signer types